Gemfile

```diff
@@ -32,6 +32,8 @@ gem 'jbuilder', '~> 2.5'
 
 gem 'govuk_admin_template'
 gem 'govuk_frontend_toolkit', :git => "https://github.com/alphagov/govuk_frontend_toolkit_gem.git", :submodules => true
+gem 'govuk-lint'
```

Comment:
This should only be added to the development (and maybe and test) environments as we don't really want them loaded in production.
```
group :development, :test do
  gem "govuk-lint"
  gem "pry"
end
```